gems/aws-sdk-core/lib/aws-sdk-core/assume_role_credentials.rb

```diff
@@ -3,25 +3,28 @@
 require 'set'
 
 module Aws
-
-  # An auto-refreshing credential provider that works by assuming
-  # a role via {Aws::STS::Client#assume_role}.
+  # A credential provider that assumes a role via
```

Comment:
Nit: Why remove the auto-refreshing from all of these descriptions?

Reply:
It was already mentioned multiple times (I made it generically below, too)